components/Chips/examples/ChipsShapingExampleViewController.m

```diff
@@ -0,0 +1,113 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ChipsExamplesSupplemental.h"
+
+#import "MaterialChips.h"
+#import "MaterialSlider.h"
+#import "MaterialShapes.h"
+#import "MaterialShapeLibrary.h"
+
+@implementation ChipsShapingExampleViewController {
+  MDCChipView *_chipView;
+  MDCSlider *_cornerSlider;
+  MDCRectangleShapeGenerator *_rectangleShapeGenerator;
+  UISegmentedControl *_cornerStyleControl;
```

Comment:
MDCTabBar? I think we should believe our components are just as good as UIKit's. :)

Reply:
I see them as 2 different UIs, and I feel like UISegmentedControl is better suited for this example. It is not full bleed and can be placed in the middle of the screen rather than bottom or top aligned.

Reply:
Fair enough. I suppose in Material we would use a List with single selection.